pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -151,7 +151,7 @@ func NewManager(topology []cadvisorapi.Node, topologyPolicyName string, topology
 		return nil, fmt.Errorf("cannot discover NUMA topology: %w", err)
 	}
 
-	if topologyPolicyName != PolicyNone && len(numaInfo.Nodes) > opts.MaxAllowableNUMANodes {
```

Comment:
not needed since L137 already ensures this will always be true

Reply:
correct. We shortcircuit at the beginning of the function, so if we got past line 141 then `topologyPolicyName` is guaranteed to be `!= PolicyNone`